src/libraries/System.Collections.Immutable/src/System/Collections/Immutable/ImmutableArray_1.Builder.cs

```diff
@@ -705,6 +732,16 @@ public void Sort(int index, int count, IComparer<T>? comparer)
                 }
             }
 
+            /// <summary>
+            /// Copies the current contents to the specified <see cref="Span{T}"/>.
+            /// </summary>
+            /// <param name="destination">The <see cref="Span{T}"/> to copy to.</param>
+            public void CopyTo(Span<T> destination)
+            {
+                Requires.Range(this.Count <= destination.Length, nameof(destination));
+                new ReadOnlySpan<T>(_elements).CopyTo(destination);
```

Comment:
Doesn't _this span need to be sliced to _count?
```suggestion
                new ReadOnlySpan<T>(_elements, 0, _count).CopyTo(destination);
```

Reply:
Fixed, thanks !